spec/requests/cookies_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe CookiesController, type: :request do
```

Comment:
There are other aspects of the cookies controller which I'm not testing here (some of the actual behaviour of the banner), but I thought that was out of scope for this ticket.